about/updates/roadmap/FY23Q1-roadmap.md

```diff
@@ -0,0 +1,72 @@
+---
+layout: page
+title: Search.gov Roadmap Fiscal Year 2023 Quarter 1
+crumbname: Roadmap FY 2023 Q1
+category: about
+tags: roadmap
+breadcrumbs:
+  - label: Home
+    url: /
+  - label: About
+sidenav: about_us
+---
+
+## Overview
+
+At Search.gov we aim to provide an excellent public search experience, a self-service search administration experience, and a rock-solid secure and compliant infrastructure to support it all, while keeping costs down across the government.
+
+We know it's helpful for our partners at federal agencies to know what's coming up next, to support planning for your own web platforms. 
+
+This roadmap shows what we are working on now, and what we plan to be working after that. We'll update this roadmap quarterly.
+
+
+## FY 2023 Q1: October&ndash;December 2022
+
+### Now
+
+* **Faceted Search &mdash; Results API Support**
+  * We will work on supporting the faceted search functionality in our Results API.
+* **Javascript Indexing &mdash; Production Release**
+  * We will be getting our Javascript indexing feature tested and ready for production.
+* **Search Engine Results Page (SERP) Redesign &mdash; Investigations**
+  * We will begin scoping the work required to upgrade our frontend technology in support of this redesign.
+* **Version Upgrades**
+  * We will be upgrading our ElasticSearch, Ruby, Rails, and Redis versions to keep them up-to-date and compliant. 
```

Comment:
oops number disagreement "upgrading our Ruby version to keep them"

Reply:
> We will be upgrading our ElasticSearch, Ruby, Rails, and Redis versions to keep them up-to-date and compliant. 

Versions are plural, shouldn't it be them?